docs/site/content/docs/latest/support-matrix.md

```diff
@@ -49,3 +49,35 @@ Mac and Windows: In Docker Desktop, select Preferences > Resources > Advanced
 ## Supported Kubernetes Versions
 
 Tanzu Community Edition supports the following Kubernetes versions: `1.21.2, 1.20.8, 1.19.12`
+
+## Check and set the cgroup
+
+1. Check the cgroup by running the following command:
+
+    ```sh
+    docker info | grep -i cgroup
+    ```
+
+    You should see the following output:
+
+    ```sh
+    Cgroup Driver: cgroupfs
+    Cgroup Version: 1
+    ```
+
+1. If you see cgroup version 2,  we recommend the following:
+   * **Mac**: Run the following compatible version of
+   Docker Desktop: [Docker Desktop
+   4.2.0](https://docs.docker.com/desktop/mac/release-notes/#docker-desktop-420).
+   * **Windows**: Run the following compatible version of [Docker Desktop
+   4.2.0](https://docs.docker.com/desktop/mac/release-notes/#docker-desktop-420)
```

Comment:
Link should be https://docs.docker.com/desktop/windows/release-notes/#docker-desktop-420

Reply:
good catch, fixed now